crates/polars-io/src/csv/parser.rs

```diff
@@ -429,6 +429,16 @@ pub(super) fn parse_lines<'a>(
                 Some((mut field, needs_escaping)) => {
                     let field_len = field.len();
 
+                    if needs_escaping && !ignore_errors {
```

Comment:
I don't want an extra branch in the hot loop. This will slow down parsing well formed csv files.